### ISSUE TITLE
Freeform polygon improvements

### ISSUE THREAD

**Issue by jakebanderson321** [anderson-optimization/frontend-documentation#303]
### Subject of the issue
Here are a handful of improvements to make on the freeform polygon search.

Once the search is completed, the polygon that was created to do the search should be deleted. Currently, the polygon remains.

The user currently has to refresh to display the new parcels. These should pull-in in real time as the do with the standard parcel radius search.

The optional "Radius" input box is confusing for this type of search. Don't display this/provide this option. Just show the acreage parameter.

**Comment by eanderson4** [comment]
are you sure you want the polygon deleted.  perhaps it should be renamed and deemphasized as it leaves a record of what has been searched.

I'll break it into two tasks, one for polygons and one for lines.  the polygons i'll remove radius parameter

**Comment by eanderson4** [comment]
For this, I think we should allow the task to return a drawing object that can be used to update the input drawing in the event.

This way the task can be responsible for changing style to deemphasize and add notes to explain details of the parcel search.

**Comment by jakebanderson321** [comment]
Good points. Yea, let's just make sure it's underneath the polygons so users can click the polygons and make the design coloring fairly minimal.

**Comment by eanderson4** [comment]
I did a proof of concept in a branch for next release.  Task output can now control styling, including polygon ordering, so it should land behind every other polygon.  Also it changes name and notes to include info on the search

Probably a little more fine tuning still but seems like decent bindings for time being.

I updated `Reportall Polygon search` on our `task-basic` team if you want to test on the branch.  Otherwise we can try to merge in sometime for next release.

![image](https://user-images.githubusercontent.com/5291870/69454103-a6666900-0d22-11ea-9c0e-368bb8e5967e.png)

![image](https://user-images.githubusercontent.com/5291870/69454164-c564fb00-0d22-11ea-86d9-0a4c6976f8e4.png)
